dist/origin-web-common.js

```diff
@@ -5787,6 +5787,34 @@ angular.module('openshiftCommonServices')
 });
 ;'use strict';
 
+angular.module("openshiftCommonServices")
+  .service("VersionsService", function(){
+    var compare = function(v1, v2, newestFirst) {
+      v1 = v1 || '';
+      v2 = v2 || '';
+      try {
+        // compareVersions will sort via semver and throw an error if one of
+        // the values is not a version string.
+        var result = window.compareVersions(v1, v2);
+        return newestFirst ? result * -1 : result;
+      } catch (e) {
+        // One of the values is not a version string. Fall back to string comparison.
+        // Numbers will be sorted higher by localeCompare.
+        return v1.localeCompare(v2);
+      }
+    };
+
```

Comment:
Quick Q ~ What do we get out of `rcompare`, assuming it means `reverseCompare`, correct?

Reply:
Yeah, the intention is that you can pass it to Arrays.sort. If you have to pass a 3rd param, that doesn't work.

Let me refactor this a bit to make it so compare and rcompare both take just the v1 and v2 params.

Reply:
Nice, this looks good to me!